xbmc/platform/darwin/ios/XBMCController.mm

```diff
@@ -30,7 +32,13 @@
 #include "utils/URIUtils.h"
 #include "utils/log.h"
 #include "windowing/XBMC_events.h"
-#include "windowing/ios/WinSystemIOS.h"
+
+#import "platform/darwin/NSLogDebugHelpers.h"
```

Comment:
use `CLog::Log` instead

Reply:
I've gone through all this on the tvos side of things, Because of the extent, its probably better to do that change in a separate PR across all ios source. For now, i'll just add the NSLogDebugHelper header.

Reply:
agree